tracer/contrib/tracegrpc/grpc.go

```diff
@@ -80,20 +80,12 @@ func serverSpan(t *tracer.Tracer, ctx context.Context, method string) *tracer.Sp
 	return span
 }
 
-func parseMethod(method string) (service, resource string) {
-
-	start := 0
-	if len(method) > 0 && method[0] == '/' {
-		start = 1
-	}
-
+func parseMethod(method string) (resource string) {
 	if idx := strings.LastIndexByte(method, '/'); idx > 0 {
-		service = method[start:idx]
-		method = method[idx+1:]
-		return service, method
+		return method[idx+1:]
```

Comment:
If we put all behind the same service, I guess we could keep the entire `method` as a resource name.